installation-bundle/src/Database/Installer.php

```diff
@@ -163,6 +163,7 @@ public function compileCommands(): void
 
                             case 0 === strncmp($part, 'CHANGE ', 7):
                             case 0 === strncmp($part, 'RENAME ', 7):
+                            case 0 === strncmp($part, 'MODIFY ', 7):
```

Comment:
```suggestion
                            case 0 === strncmp($part, 'MODIFY ', 7):
                            case 0 === strncmp($part, 'ALTER ', 6):
```
Should we also allow `ALTER` for queries like `ALTER TABLE tbl ALTER col SET DEFAULT ''`?

Reply:
I haven't had this need yet so I don't know about possible side effects and I cannot really test it but if you are sure that works... :)

Reply:
I’m not sure ☺️
Feel free to ignore.